test/reporters/verbose.js

```diff
@@ -89,6 +89,20 @@ test('don\'t display test title if there is only one anonymous test', function (
 	t.end();
 });
 
+test('known failure test', function (t) {
+	var reporter = createReporter();
+
+	var actualOutput = reporter.test({
+		title: 'known failure',
+		failing: true
+	}, createRunStatus());
+
+	var expectedOutput = '  ' + chalk.red(figures.tick) + ' ' + chalk.red('known failure');
+
+	t.is(actualOutput, expectedOutput);
```

Comment:
Change the assertion to

`t.is(JSON.stringify(actual), JSON.stringify(expected)`